packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -5024,6 +5035,22 @@ export interface StartCaptionsButtonStrings {
     tooltipOnContent: string;
 }
 
+// @beta
+export const StartRealTimeTextButton: (props: StartRealTimeTextButtonProps) => JSX.Element;
+
+// @beta (undocumented)
```

Comment:
if this is a public component lets add a comment here

Reply:
Updated